src/models/TraktItem.ts

```diff
@@ -20,7 +20,7 @@ export interface TraktItemBase {
 
 export interface SavedTraktItem extends TraktItemBase {
 	releaseDate?: number;
-	watchedAt?: number;
+	watchedAt?: number | null;
```

Comment:
Do we need to keep both undefined and null?

Reply:
Yes, because there's a distinction between the two:

* `undefined` means that we have not tried to retrieve the data yet
* `null` means that we have tried to retrieve the data and it was unavailable or nonexistent

Based on this, we can either show "Loading history..." if it's `undefined` or "Not Watched" if it's `null`.

I think this works best. This distinction is also used in many other parts of the extension that involve retrieving data from somewhere, like when having to retrieve suggestions or images.

Reply:
Ahh! Thanks for clarifying!